src/Cli/dotnet/commands/dotnet-test/IPC/IClient.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace Microsoft.DotNet.Tools.Test;
+
+internal interface IClient : IDisposable
+#if NETCOREAPP
+#pragma warning disable SA1001 // Commas should be spaced correctly
+    , IAsyncDisposable
```

Comment:
I think you won't have to disable the warnings if you reverse the order of these interfaces, i.e.,
IClient :
#if NETCOREAPP
IAsyncDisposable,
#endif
IDisposable

Reply:
@mariam-abdulla pls give it a try looks a good idea! thanks